dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9.12-buster as base
+WORKDIR /app
+COPY . /app/
+RUN pip install poetry
+RUN poetry install
```

Comment:
If you want to make this a little quicker when you're developing, one trick is to split out the copy stages so that you can cache the `poetry install` step since this is the slowest stage:
```
RUN pip install poetry
COPY pyproject.toml poetry.lock /app/
RUN poetry install
COPY . /app/
```